Tests/UnitTests/Core/Vertexing/AdaptiveMultiVertexFinderTests.cpp

```diff
@@ -496,7 +498,7 @@ BOOST_AUTO_TEST_CASE(adaptive_multi_vertex_finder_grid_seed_finder_test) {
 
   Fitter fitter(fitterCfg);
 
-  // using SeedFinder = TrackDensityVertexFinder<Fitter, GaussianTrackDensity>;
+  // using SeedFinder = TrackDensityVertexFinder<Fitter, TrackDensity>;
   using SeedFinder = GridDensityVertexFinder<2000, 35>;
```

Comment:
Is this always given to be 200,35?

Reply:
No, this is just one example with more or less random numbers / educated guesses for this unit test.

Reply:
Ah, sorry I missed that this is in the UnitTests. Fair enough then.